avocado/utils/archive.py

```diff
@@ -31,37 +36,38 @@ class ArchiveException(Exception):
     pass
 
 
-class _WrapLZMA(object):
+if LZMA_SUPPORTED:
```

Comment:
We can drop this `if` line, to create class. On line 28, after `LZMA_SUPPORTED = False`, define `lzma = None`, so it will be safe to remove the conditional.